### PR TITLE
Fix missing links and unused files

### LINKED ISSUE #33: Update URLs from old website
After https://github.com/ICESAT-2HackWeek/icesat-2hackweek.github.io/pull/16 , some images are no longer rendering because they point to the previous website! 

Screen Shot 2022-03-02 at 1 36 15 PM

### PR DESCRIPTION
**Pull request by scottyhq**
Closes #33

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,8 +16,8 @@ Ready to contribute? Here's a quick guide
 1. Clone your fork locally:
 
     ```sh
-    git clone {{website_url}}.git
-    cd jupyterbook-template
+    git clone https://github.com/ICESAT-2HackWeek/website2022.git
+    cd website2022
```
Comment (JessicaS11) [inline CONTRIBUTING.md:20]:
```suggestion
    git clone {{repository.url}}.git
    cd {{book_repo}}
```
Assuming you can use dot syntax to access the url from the [_config.yaml](https://github.com/ICESAT-2HackWeek/website2022/blob/imagelinks/book/_config.yml).

Comment (scottyhq) [reply CONTRIBUTING.md:20]: i believe the problem is this file is outside 'book' folder and therefore jupyterbook doesn't know about it. Could remedy this via cookiecutter and https://github.com/uwhackweek/jupyterbook-template/issues/84 in the future

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ Ready to contribute? Here's a quick guide
     git push origin name-of-your-bugfix-or-feature
     ```
 
-1. Open a pull request through the GitHub website: {{website_url}}
+1. Open a pull request through the GitHub website: https://github.com/ICESAT-2HackWeek/website2022
```
[the remark this answers is not in the document]
Comment (scottyhq) [reply CONTRIBUTING.md:56]: added a comment https://github.com/uwhackweek/jupyterbook-template/issues/84#issuecomment-1058309794

**Comment by scottyhq** [comment]
@JessicaS11 I linked to an existing issue in the template repository, ok to merge?

**Review by JessicaS11 — APPROVED** [review]

**Pull request merged**